spec/services/charts_layout_service_spec.rb

```diff
@@ -19,6 +19,8 @@
     end
 
     it "returns layout for fname if specific class does not exist" do
+      allow(host_redhat).to receive(:cpu_percent_available?).and_return(true)
+      allow(host_redhat).to receive(:cpu_mhz_available?).and_return(true)
```

Comment:
NOTE this test is just checking the layout that is loaded, the describe below tests the different `applies_to_method` for vms so I added the Host `applies_to_method` tests down there